src/main/java/com/mitchej123/hodgepodge/client/HodgepodgeClient.java

```diff
@@ -67,6 +68,11 @@ public static void postInit() {
             meta.description = "This IC2 Addon allows you to simulate the CropBreeding.";
             meta.autogenerated = false;
         }
+
+        if (Compat.isBiomesOPlentyPresent()) {
+            // removes the popup that BOP shows on first world gen
```

Comment:
This should really only be done in gtnh. Hodgepodge is used outside of the pack and I mostly support that.  A config would also be nice